### PR TITLE
New medkit and trauma pouch sprites

### PR DESCRIPTION
**Pull request by nauticall**
Gives the medkits and trauma pouches yet another fresh coat of paint. They now look sleek and nice and have this cool stripe on them that tells you what they are.

But it's not only the outside -- the pouches inside several of the kits now look nicer too! Subjectively speaking.

Thirdly, the Horizon's medical bay storage room has been mapped to accommodate the new medkits and stack them all in a lovely, tidy, perfectionist-pleasing manner. You're welcome, doctors. 

## Screenshots
image
_Not pictured ingame: the combat kit, because I forgot to update it in this instance and am too lazy to reboot the server just for that._

image
_DMI file stuff. Inhands included._

image
_What can I say except you're welcome? For the kits, the sprites, the... sky._

image
_The rainbow of new pouches. Yes, I tore one open to show that too. Not sorry._

### CODE REVIEW

**Comment by nauticall** [comment]
!review

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 1]
```diff
@@ -146,7 +140,6 @@
 
 /obj/item/storage/firstaid/radiation/Initialize()
```
Comment (NonQueueingMatt) [inline code/game/objects/items/weapons/storage/firstaid.dm:141]: need to delete these Initialize() procs altogether if all they do is call `..()`

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 2]
```diff
@@ -133,7 +128,6 @@
 
 /obj/item/storage/firstaid/trauma/fill()
```
Comment (NonQueueingMatt) [inline code/game/objects/items/weapons/storage/firstaid.dm:129]: need to delete these fill() procs altogether if all they do is call `..()`

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 3]
```diff
@@ -80,7 +77,6 @@
 
 /obj/item/storage/firstaid/adv/fill()
 	. = ..()
```
Comment (NonQueueingMatt) [inline code/game/objects/items/weapons/storage/firstaid.dm:79]: need to delete these fill() procs altogether if all they do is call `..()`

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 4]
```diff
@@ -63,7 +61,6 @@
 
 /obj/item/storage/firstaid/o2/fill()
 	. = ..()
```
Comment (NonQueueingMatt) [inline code/game/objects/items/weapons/storage/firstaid.dm:63]: need to delete these fill() procs altogether if all they do is call `..()`

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 5]
```diff
@@ -37,20 +37,18 @@
...
-	icon_state = "antitoxinfirstaid"
-	item_state = "antitoxinfirstaid"
+	icon_state = "antitoxfirstaid"
+	item_state = "antitoxfirstaid"
 	starts_with = list(
 		/obj/item/storage/box/fancy/med_pouch/toxin = 4
 	)
 
 /obj/item/storage/firstaid/toxin/fill()
 	. = ..()
```
Comment (NonQueueingMatt) [inline code/game/objects/items/weapons/storage/firstaid.dm:51]: need to delete these fill() procs altogether if all they do is call `..()`

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Review by niennab — APPROVED** [review]

**Review by alsoandanswer — COMMENTED** [review]
Sorry, this resprite isn't really needed. 

I *just* resprited these medkits not too long ago. 

Please check if the .dmi file has been edited recently and if so, ask them
(especially if they're part of the active developer team!)

This also breaks Medbots, since they rely on the boxy sprite as their base icon.

**Comment by Generalcamo** [comment]
I'd recommend splitting out the medpouches to a new PR. Those were a direct port from Bay, and the resprite would be accepted quickly.

**Comment by nauticall** [comment]
ah well! was good practice anyway, gonna PR the pouches/descs separately then

**Comment by nauticall** [comment]
image
New-new kits! I've been asked to keep the existing ones' silhouette and shape consistent so that they don't screw up the medbots/map placements and have also repurposed the existing ones into "large" first-aid kits with _lots_ of treatments and more advanced medication than the standard model.

Mapping these large kits in is a biiit out of scope for this PR so that'll come later, both to avoid conflicts and so I don't end up having this PR stuck in merge hell because of the mapping stuff.

**Review by NonQueueingMatt — APPROVED** [review]

**Review by alsoandanswer — APPROVED** [review]

**Pull request merged**